### PR TITLE
Awesome Books with ES6

### PR DESCRIPTION
**Pull request by Victor-chiemerie**
> ### Highlights 🥇
> **Awesome books: with ES6`**
> 
> **Learning objectives**
> 
> * Use ES6 modules to write modular JavaScript.
> * Learn how to use proper ES6 syntax.
> * Understand improvements that ES6 brings to JavaScript.
> * Use npm as software packages system.
> 
> **What I Have Done:**
> 
> * Added linters. The [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
> * Create a collection that keeps a list of books.
> * Create a function to add a new book to the collection.
> * When a user clicks the "Add" button, a new book is added to the collection
> * When a user clicks the "Remove" button,the correct book is removed from the collection.
> * The data is stored in the browser's memory using localStorage
> * I use JavaScript classes instead of objects and arrays.
> * I used objects and arrays inside classes.
> * I create class methods to add and remove books
> * I ensured that when you click on any of the links, the main area changes to display only one of the three content sections.
> * Use ES6 syntax.
> * I divided my code into modules. and save each module in a separate file in the modules dir.
> * I ensure that all functionalities of the app are without errors.
> * I refactor my methods and functions to arrow functions.
> * I use let and const in a correct way.
> * I display the current date below the navigation bar.
> 
> **Followed Rules:**
> 
> * Made sure [no linter errors](https://github.com/microverseinc/linters-config)
> * Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
> * Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
> * Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
> * Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: modules/showList.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  listBtn.addEventListener('click', () => {
+    list.classList.remove('hidden');
+    addNew.classList.add('hidden');
+    contact.classList.add('hidden');
+    listBtn.style.color = 'rgb(55, 135, 172)';
+    addNewBtn.style.color = 'black';
+    contactBtn.style.color = 'black';
+  });
+}
```
Comment (leonardodiasb) [inline modules/showList.js]: - [x] Kindly use the ES6 arrow syntax for functions in your code to meet the project requirements.

File: modules/showContact.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const contactBtn = document.querySelector('.contactBtn');
+
+  contactBtn.addEventListener('click', () => {
+    list.classList.add('hidden');
+    addNew.classList.add('hidden');
+    contact.classList.remove('hidden');
+    listBtn.style.color = 'black';
+    addNewBtn.style.color = 'black';
+    contactBtn.style.color = 'rgb(55, 135, 172)';
+  });
```
Comment (leonardodiasb) [inline modules/showContact.js]: - [x] Kindly use the ES6 arrow syntax for functions in your code to meet the project requirements.

File: modules/showAddNew.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  const contactBtn = document.querySelector('.contactBtn');
+
+  addNewBtn.addEventListener('click', () => {
+    list.classList.add('hidden');
+    addNew.classList.remove('hidden');
+    contact.classList.add('hidden');
+    listBtn.style.color = 'black';
+    addNewBtn.style.color = 'rgb(55, 135, 172)';
+    contactBtn.style.color = 'black';
+  });
```
Comment (leonardodiasb) [inline modules/showAddNew.js]: - [x] Kindly use the ES6 arrow syntax for functions in your code to meet the project requirements.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+const booklist = document.querySelector('#booklist');
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const buttonAdd = document.getElementById('add_button');
+
+const obj = new Library();
+
+function printBooks() {
+  const shelf = JSON.parse(localStorage.getItem('shelf')) || [];
+  let innerhtml = '';
```
Comment (leonardodiasb) [inline index.js]: - [x] Kindly use the ES6 arrow syntax for functions in your code to meet the project requirements.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Victor-chiemerie, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Clean code and organized. ✔️
 - No linter errors. ✔️ 
 - Professional PR and README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Victor-chiemerie 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of Es6 arrow functions ✔️ 
- Good use of `let` and `const` keywords ✔️

### Optional Suggestion
- Kindly include a _**Future Features**_ section that contains 1 - 3 features that can be implemented in the future. This will serve as guidelines to other developers to let them know how they can contribute to the project. It could also serve as a reference point for you if you want to make changes to the code in the future. 👍 
- In your _**MIT.md**_ file, kindly replaces the phrase **_[YOUR NAME]_** with your actual name and all phrases **_[APP TYPE]_** with the name of your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**